Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,40 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string firstName;
+            string lastName;
+
+            int age;
+            double weight;
```

Comment:
Introducing variables like this will force a programmer to memorize the type until it's first usage. To avoid confusion you should introduce variables as late as possible so the programmer can focus on what is needed rather than what will be used somewhere in the future.

Reply:
In this situation though, wouldn't moving the variables put them inside the for loop causing them to be declared twice at run time? I'm not sure when this is something I need to worry about.